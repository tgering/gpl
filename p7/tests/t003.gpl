int i = 42;
double x = 3.14159;
string str = "hello world";

on space
{
    print("i = " + i);
    print("x = " + x);
    print("str = " + str);
}
