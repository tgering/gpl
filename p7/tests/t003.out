gpl.cpp::main()
  input file(tests/t003.gpl)
  random seed(42)
  read_keypresses_from_standard_input(true)
  dump_pixels(false)
  symbol_table(true)
  graphics(true)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().


Printing the symbol table from main()
int i = 42
string str = "hello world"
double x = 3.14159
gpl.cpp::main() Calling window->initialize().


gpl.cpp::main() Passing control to window->main_loop().
print[7]: i = 42
print[8]: x = 3.14159
print[9]: str = hello world

User quit program. Printing the symbol table.
int i = 42
string str = "hello world"
double x = 3.14159
